graylog-storage-elasticsearch6/src/test/resources/org/graylog/storage/elasticsearch6/views/everySearchTypeGeneratesOneESQuery.request.ndjson

```diff
@@ -1,4 +1,4 @@
 {"index" : "", "type" : "message", "ignore_unavailable" : "false", "allow_no_indices" : "false"}
-{  "from" : 0,  "size" : 0,  "query" : {    "bool" : {      "must" : [        {          "bool" : {            "filter" : [              {                "match_all" : {                  "boost" : 1.0                }              }            ],            "disable_coord" : false,            "adjust_pure_negative" : true,            "boost" : 1.0          }        },        {          "range" : {            "timestamp" : {              "from" : "2018-08-23 08:02:00.247",              "to" : "2018-08-23 08:07:00.252",              "include_lower" : true,              "include_upper" : false,              "boost" : 1.0            }          }        },        {          "terms" : {            "streams" : [ ],            "boost" : 1.0          }        }      ],      "disable_coord" : false,      "adjust_pure_negative" : true,      "boost" : 1.0    }  },  "aggregations" : {    "pivot1-series-avg(field1)" : {      "avg" : {        "field" : "field1"      }    },    "timestamp-min" : {      "min" : {        "field" : "timestamp"      }    },    "timestamp-max" : {      "max" : {        "field" : "timestamp"      }    }  }}
+{"from":0,"size":0,"query":{"bool":{"must":[{"bool":{"filter":[{"match_all":{"boost":1.0}}],"adjust_pure_negative":true,"boost":1.0}},{"range":{"timestamp":{"from":"2018-08-23 08:02:00.247","to":"2018-08-23 08:07:00.252","include_lower":true,"include_upper":false,"boost":1.0}}},{"terms":{"streams":[],"boost":1.0}}],"adjust_pure_negative":true,"boost":1.0}},"aggregations":{"pivot1-series-avg(field1)":{"avg":{"field":"field1"}},"timestamp-min":{"min":{"field":"timestamp"}},"timestamp-max":{"max":{"field":"timestamp"}}}}
```

Comment:
It's really hard to see the difference, but the change is rather minor, only ` "disable_coord": false` added automatically when the query is serialized in the ES6 version. Similar minor and unrelated changes are in the other 2 test resource files.